scripts/cipublish

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
The publishing story needs to be improved in general: at the moment the images are publishing to my Docker Hub.